### PR TITLE
Fix: [Violation] Added non-passive event listener to a scroll-blocking 'wheel' event

### PR DESCRIPTION
**Pull request by tienifr**
### Fixed Issues

$ https://github.com/Expensify/App/issues/26990
PROPOSAL: https://github.com/Expensify/App/issues/26990#issuecomment-1714914733

### Tests

1. Open Expensify ND
2. Verify that the warning: `[Violation] Added non-passive event listener to a scroll-blocking 'wheel' event` does not appear in the JS console

- [x] Verify that no errors appear in the JS console

### Screenshots/Videos
<details>
<summary>Web</summary>

https://github.com/Expensify/react-native-web/assets/113963320/b78eba08-b679-40b5-bc5a-be074e408e59

</details>

### CODE REVIEW

**Comment by tienifr** [comment]
@thesahindia This one is ready for review!

File: packages/react-native-web/src/modules/canUsePassive/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/**
+ * Allows us to identify whether the browser supports passive event listener.
+ * Because older browsers will interpret any object in the 3rd argument of an event listener as capture=true.
```
Comment (thesahindia) [inline packages/react-native-web/src/modules/canUsePassive/index.js]: NAB: Don't think this line is necessary
```suggestion

```

Comment (tienifr) [reply packages/react-native-web/src/modules/canUsePassive/index.js]: Updated!

**Comment by tienifr** [comment]
@thesahindia bump.

**Comment by thesahindia** [comment]
> @thesahindia bump.

@tienifr, can you help me with the steps that I need to get the warning? I am not getting it. Was there any specific step?

**Comment by getusha** [comment]
@thesahindia @tienifr can we raise the PR upstream and apply the patch? it will be helpful since this fork is likely to be removed.

**Comment by tienifr** [comment]
@getusha Thanks a lot! Can you give us the context on the removal of this fork?

@thesahindia What's your opinion?

**Comment by getusha** [comment]
@tienifr we will be using `react-native-web` directly instead of this fork https://github.com/Expensify/App/pull/24482

**Comment by thesahindia** [comment]
>@thesahindia @tienifr can we raise the PR upstream and apply the patch? it will be helpful since this fork is likely to be removed.

That makes sense to me!

**Pull request closed without merging**